packages/ListBox/src/ListBox.js

```diff
@@ -0,0 +1,235 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Box from "./components/Box";
+import Content from "./components/Content";
+import Filter from "./components/Filter";
+import List from "./components/List";
+import NoResults from "./components/NoResults";
+import Options from "./components/Options";
+import Popover from "./components/Popover";
+import Trigger from "./components/Trigger";
+import useListBox from "./useListBox";
+import handleImperative from "./imperative";
+import * as effects from "./effects";
+
+export const propTypes = {
+  /** Child of type <ListBox.Option /> */
+  children: PropTypes.node,
+
+  /** Turn on the input filter for the options */
+  hasFilter: PropTypes.bool,
+
+  /** Turn on/off the clear button at the right side of the Trigger */
+  hasClearButton: PropTypes.bool,
+
+  /** Indicate which is the height for the options container */
+  height: PropTypes.number,
+
+  /** [Advance] instead of marking the option as checked/unchecked will toggle the option between visible and hidden */
+  hideOptionOnSelected: PropTypes.bool,
+
+  /** Disable the entire ListBox */
+  isDisabled: PropTypes.bool,
+
+  /** Let the user to select multiple options at same time */
+  isMulti: PropTypes.bool,
+
+  /** This options will display the listbox without the Popover */
+  isInline: PropTypes.bool,
+
+  /** When true the ListBox will try to focus to the options container asap the
+  popover is open */
+  isPopoverEager: PropTypes.bool,
+
+  /** Indicates if the popover is visible */
+  isPopoverOpen: PropTypes.bool,
+
+  /** Message to be display once the filtering process doesn't find a match */
+  hasNotResultsMessage: PropTypes.node,
```

Comment:
`hasNotResultsMessage` should be `hasNoResultsMessage`.
Even better, I prefer the name `noResultsMessage`, without the `has` — as that makes me think its boolean.